### ISSUE TITLE
Duplicate volumes in version 107.39.4

### ISSUE THREAD

**Issue by mouliveera** [jfrog/charts#1650]
**Is this a request for help?**:
Yes
---

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):
## BUG REPORT

**Version of Helm and Kubernetes**:
Helm: v3.8.2
K8s: v1.24.0

**Which chart**:
https://artifacthub.io/packages/helm/jfrog/artifactory/107.39.4

**What happened**:
Duplicate volumes defines:
https://artifacthub.io/packages/helm/jfrog/artifactory/107.39.4?modal=template&template=artifactory-statefulset.yaml
Line numbers:
1167-1176

1204-1212

**What you expected to happen**:
Remove the duplicates in latest code

**How to reproduce it** (as minimally and precisely as possible):
Run 
helm template -f values.yaml

**Anything else we need to know**:
No thanks

**Comment by mouliveera** [comment]
Also with previous version observed init containers are failing with some OSI layer issues.
```
Error: Failed to create containers task: failed to create shim: OCI runtime create failed: container_linux.go:380: starting container process caused: exec: "bash": executable file not found in $PATH: unknown
```

**Comment by dennishellerdigatus** [comment]
also experiencing the issue with the duplicate volumes, preventing us to upgrade to 107.39.4

**Comment by itssiri** [comment]
Thanks for reporting the issue. Will check and fix this in upcoming  release.

**Comment by chukka** [comment]
Fixed via #1682
